torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -26,17 +29,17 @@ def horizontal_flip_bounding_box(
     ).view(shape)
 
 
-def resize_image(
+def resize_image_tensor(
     image: torch.Tensor,
     size: List[int],
-    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    interpolation: str = "nearest",
```

Comment:
First of all we should use bilinear as the default, not nearest. This is already the default on functional_tensor.resize.

Second I believe we should move towards Enums here. I think one reason we don't support them here is for JIT scriptability but now enums are scriptable. I would change it here and perhaps on a separate PR, I would update the tensor functional to use enums (it is currently considered private so we can do it). Thoughts?

Reply:
We have warnings like 

https://github.com/pytorch/vision/blob/761d09f1c3dd94ffe489f2c08a72a719f996a93d/torchvision/transforms/transforms.py#L255-L261

since `0.9`. If we strictly stick to the new [BC policy](https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy), we can only remove if the warning includes the version this is going to be removed. This is why we merged #5387 and others before this release to be able to remove with `0.14`. If we also use this scheme here, we could only remove in `0.15`.

However, the spirit of the BC policy is to keep the behavior for two releases or 180 days in the nightlies. Since we are way past this, I think we can go for the removal here given that it would simplify the implementation quite a bit. cc @NicolasHug @mruberry to make sure.

Reply:
@pmeier Dont take what I say as disagreement on the deprecation policy stuff. It's unrelated.

What I say here is that the `functional_tensor.py` file was considered private. This was done several releases back. So due to this, you can do all changes as you want without BC concerns.

Reply:
I would also consider all BC-breaking changes made before the latest policy as "grandfathered into" whatever old policies (or lack of old policies) existed. That said, when those changes are public they could be documented following the new policy's suggestions.